core/shared/platform/rt-thread/platform_internal.h

```diff
@@ -0,0 +1,34 @@
+//
+// Created by alvis on 2020/12/14.
+//
+
+#ifndef RTTHREAD_PLATFORM_INTERNAL_H
+#define RTTHREAD_PLATFORM_INTERNAL_H
+
+#include <rtthread.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <stdint.h>
+#include <ctype.h>
+
+#ifndef BUILD_TARGET
```

Comment:
Not sure whether we should define BUILD_TARGET here, normally we define BUILD_TARGET_XXX in build-scripts/config_common.cmake by parsing the WAMR_BUILD_TARGET=XXX passed by cmake variable (cmake -DWAMR_BUILD_TARGET=XXX).

And had better define the BH_PLATFORM_XXX macro in this file, so other files can use the macro, e.g. [wasm_native.c](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/common/wasm_native.c#L10):
#ifndef BUILD_PLATFORM_RT_THREAD
#define BUILD_PLATFORM_RT_THREAD
#endif

Reply:
sorry, this definiation was used in the past version and I forgot to delete it, I just remove it from `rt-thread/platform_internal.h`.